clients/ui/frontend/src/concepts/modelRegistry/utils.ts

```diff
@@ -26,3 +27,61 @@ export const modelSourcePropertiesToCatalogParams = (
     tag: properties.modelSourceId,
   };
 };
+
+export const catalogParamsToModelSourceProperties = (
+  params: CatalogModelDetailsParams,
+): ModelSourceProperties => ({
+  modelSourceKind: ModelSourceKind.CATALOG,
+  modelSourceClass: params.sourceName || '',
+  modelSourceGroup: params.repositoryName || '',
+  modelSourceName: params.modelName,
+  modelSourceId: params.tag,
+});
+
+export const modelSourcePropertiesToCustomProperties = (
+  properties: ModelSourceProperties,
+): ModelRegistryCustomProperties => {
+  const customProperties: ModelRegistryCustomProperties = {};
+
+  if (properties.modelSourceKind === ModelSourceKind.CATALOG) {
+    customProperties['Registered from'] = {
+      // eslint-disable-next-line camelcase
+      string_value: 'Model catalog',
+      metadataType: ModelRegistryMetadataType.STRING,
+    };
+
+    if (properties.modelSourceName) {
+      customProperties['Source model'] = {
+        // eslint-disable-next-line camelcase
+        string_value: properties.modelSourceName,
+        metadataType: ModelRegistryMetadataType.STRING,
+      };
+    }
+
+    if (properties.modelSourceId) {
+      customProperties['Source model version'] = {
+        // eslint-disable-next-line camelcase
+        string_value: properties.modelSourceId,
+        metadataType: ModelRegistryMetadataType.STRING,
+      };
+    }
+
+    if (properties.modelSourceClass) {
+      customProperties.Provider = {
+        // eslint-disable-next-line camelcase
+        string_value: properties.modelSourceClass,
+        metadataType: ModelRegistryMetadataType.STRING,
+      };
+    }
+
+    if (properties.modelSourceGroup) {
+      customProperties['Source model id'] = {
+        // eslint-disable-next-line camelcase
+        string_value: properties.modelSourceGroup,
+        metadataType: ModelRegistryMetadataType.STRING,
+      };
+    }
+  }
+
+  return customProperties;
+};
```

Comment:
This isn't necessary. We do not want to use custom properties AT ALL for this feature.